src/app/Fake.DotNet.Fsdocs/Fsdocs.fs

```diff
@@ -130,13 +142,15 @@ module Fsdocs =
         System.Text.StringBuilder()
         |> StringBuilder.appendIfSome buildParams.Input (sprintf "--input %s")
         |> StringBuilder.appendIfSome buildParams.Projects (fun projects ->
-            sprintf "--projects %s" (projects |> String.concat ","))
+            sprintf "--projects %s" (projects |> String.concat " "))
```

Comment:
This wasn't working with fsdocs, it uses spaces rather than commands to list parameters.